index.html

```diff
@@ -835,12 +835,14 @@ <h3>INR 2500</h3>
                             </table>
                           </div>
                       </div>
+                      <br>
+                      <p>Want to come for the conference, but not sure how to pitch the same to your boss. Leave it to us, check out <a href ="http://www.in.pycon.org/convince-my-boss.html"><strong><u>Convince My Boss.</u></strong></a></p>
```

Comment:
Just put the link as `/convince-my-boss.html`

Reply:
Also this is not being displayed in the home page. Didn't you push this to your gh-pages branch ?

Reply:
An absolute works out great. Thanks. Also, `/` is not needed since everything is in the root directory.
Also, I forgot to push. It should be visible now.